docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+web:
+    build: .
+    command: bundle exec rails s -p 3000 -b '0.0.0.0'
+    volumes:
+        - .:/workspace
+    ports:
+        - "3000"
```

Comment:
Do you get an error? Because it works as is.

Docker version 1.13.1
docker-compose version 1.5.2
Ubuntu 16.10, kernel 4.8